### PR TITLE
Lab1

### PR DESCRIPTION
**Pull request by MilaHalko**

### CODE REVIEW

**Comment by MilaHalko** [comment]
http://laba1.milkadeluxe.fun/index.html

File: index.html [hunk 1]
```diff
@@ -1,8 +1,85 @@
 <!DOCTYPE html>
-<html lang="en">
+<html>
   <head>
-    <meta charset="UTF-8" />
-    <title></title>
+    <title>Laboratory work #1 by Mila Halko</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
```
Comment (nosovk) [inline index.html]: meta name="viewport" пропущен

File: index.html [hunk 2]
```diff
@@ -1,8 +1,85 @@
...
+    <header>
+      <div>
+        <q
+          >“Any fool can write code that a computer can understand. Good
+          programmers write code that humans can understand.” – Martin Fowler</q
+        >
+      </div>
+      <div class="top-title"><h1>HTML</h1></div>
+    </header>
+    <div class="center">
```
Comment (nosovk) [inline index.html]: use semantic tags (like article/main)

File: lab1.2.css [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+  font-family: "Izax";
+  font-size: 15px;
+  font-weight: 500;
+  line-height: 20px;
+}
+
+/* SAME COLORS */
+header,
+footer {
+  background: #0f489c;
```
Comment (nosovk) [inline lab1.2.css]: you can use css vars as values here

File: lab1.2.css [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+  align-items: center;
+  border: 5px solid var(--darkblue-color);
+  border-radius: 5px;
+  display: flex;
+  justify-content: center;
+  margin: 0 10px;
+}
+
+/* Center sides */
+.center__top,
```
Comment (nosovk) [inline lab1.2.css]: using `.center .top`, `.center .bottom`
is beat easier, especially if you use scss

File: lab1.3.css [hunk 5]
```diff
@@ -0,0 +1,212 @@
+/* BASICS */
```
Comment (nosovk) [inline lab1.3.css]: is it actually needed to make full copy of css for each page? Can we use one css with custom selectors, or split it to smaller ones?

**Comment by MilaHalko** [comment]
http://laba1.milkadeluxe.fun/lab1/index.html

**Comment by nosovk** [comment]
merge

**Pull request merged**